org.eclipse.jdt.ui/ui/org/eclipse/jdt/ui/text/java/correction/CUCorrectionProposal.java

```diff
@@ -165,10 +165,10 @@ public CUCorrectionProposal(String name, ICompilationUnit cu, TextChange change,
 
 
 	/**
-	 * @since 3.31
+	 * @since 3.32
 	 * @return the delegate
 	 */
-	protected CUCorrectionProposalCore getDelegate() {
+	public CUCorrectionProposalCore getDelegate() {
```

Comment:
I don't think you can do this.  In my tests, if I extend an external class with protected and override that protected method, if the original class changes to public, I get an error.

Reply:
Fixed using IAdaptable